src/raftstore/store/debug.rs

```diff
@@ -151,6 +152,14 @@ impl Debugger {
             Err(e) => Err(box_err!(e)),
         }
     }
+
+    pub fn compact(&self, cf: String, from: Vec<u8>, to: Vec<u8>) -> Result<()> {
+        try!(validate_db_and_cf(DB::KV, &cf));
+        let db = self.engines.kv_engine.clone();
+        CompactRunner::new(db)
```

Comment:
bad style to use CompactRunner here. 

I suggest extracting a compact function in the util/rocksdb. Btw, we may compact raft RocksDB too.